srcpkgs/vboot-utils/template

```diff
@@ -17,9 +16,7 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 do_fetch() {
-	git clone https://chromium.googlesource.com/chromiumos/platform/vboot_reference.git $pkgname-$version || true
-	cd $pkgname-$version
-	git checkout $_githash
+	git clone -b release-R${version/./-}.B https://chromium.googlesource.com/chromiumos/platform/vboot_reference.git $pkgname-$version
```

Comment:
The upstream supports tar.gz snapshots from revisions now. We should use this for fetching the binaries instead.

Also I just remembered why I chose to not depend on a named revision, like a tag or a branch. This allows the upstream to change the revisions content without any notice. With a commit hash we have a cryptographic proof of authenticity.

Reply:
I don't think that holds very well, we use tar release tarballs for probably >90% of packages, we don't have any kind of cryptographic proof of authenticity for those.

Reply:
The only defenses against this are:

- contench checksum (doesn't work for all packages)
- checking tarball signature
- comparing contents of the cached tarball with old checksum against new tarball

Reply:
@fosslinux Not true, we check the tar.gz for the right sha256 sum and we hit this issue multiple times where upstream has changed the tars after the release.

Reply:
> * comparing contents of the cached tarball with old checksum against new tarball

@ericonr Yea, I remember a few cases where I did exactly that

Reply:
Okay, It's not a proof for authenticity. But we can verify if the content got changed.

Reply:
@Gottox And for the record, I do agree with you on one point here: if we use a commit hash, we can way more easily track down the original tarball, if it isn't available from distfile caches. With just the tag, it's very hard to find the original ref.

Reply:
yes, that's true.

Reply:
Hm, ok, fair, kinda forgot about checksums :P I like your diff below, that looks good to me.